lib/kastlex.ex

```diff
@@ -3,91 +3,89 @@ defmodule Kastlex do
 
   require Logger
 
-  @anon "anonymous"
-
   def start(_type, _args) do
     import Supervisor.Spec, warn: false
 
+    # http endpoint
     endpoint = Application.fetch_env!(:kastlex, Kastlex.Endpoint)
     http = endpoint[:http]
     port = system_env("KASTLEX_HTTP_PORT", http[:port])
     Logger.info "HTTP port: #{port}"
     http = Keyword.put(http, :port, port)
     Application.put_env(:kastlex, Kastlex.Endpoint, Keyword.put(endpoint, :http, http))
+    maybe_init_https(system_env("KASTLEX_USE_HTTPS", false, &s2b/1))
+    maybe_set_secret_key_base(system_env("KASTLEX_SECRET_KEY_BASE"))
+    maybe_set_guardian_secret_key(system_env("KASTLEX_JWK_FILE"))
 
-    maybe_init_https(System.get_env("KASTLEX_USE_HTTPS"))
-    maybe_set_secret_key_base(System.get_env("KASTLEX_SECRET_KEY_BASE"))
-    maybe_set_guardian_secret_key(System.get_env("KASTLEX_JWK_FILE"))
-    kafka_endpoints = parse_endpoints(System.get_env("KASTLEX_KAFKA_CLUSTER"), [{'localhost', 9092}])
-    Logger.info "Kafka endpoints: #{inspect kafka_endpoints}"
-
+    # authentication/authorization
     permissions_file_path = system_env("KASTLEX_PERMISSIONS_FILE_PATH", "permissions.yml")
     Logger.info "Permissions file path: #{permissions_file_path}"
+    Application.put_env(:kastlex, :permissions_file_path, permissions_file_path)
     passwd_file_path = system_env("KASTLEX_PASSWD_FILE_PATH", "passwd.yml")
     Logger.info "Passwd file path: #{passwd_file_path}"
+    Application.put_env(:kastlex, :passwd_file_path, passwd_file_path)
+
+    # consumer groups
     cg_cache_dir = system_env("KASTLEX_CG_CACHE_DIR", :priv)
     Logger.info "Consumer groups cache directory: #{cg_cache_dir}"
+    Application.put_env(:kastlex, :cg_cache_dir, cg_cache_dir)
     cg_exclude_regex = system_env("KASTLEX_CG_EXCLUDE_REGEX", nil)
     maybe_log_parameter("Consumer groups exclude regexp", cg_exclude_regex)
-
-    Application.put_env(:kastlex, :permissions_file_path, permissions_file_path)
-    Application.put_env(:kastlex, :passwd_file_path, passwd_file_path)
-    Application.put_env(:kastlex, :cg_cache_dir, cg_cache_dir)
     Application.put_env(:kastlex, :cg_exclude_regex, cg_exclude_regex)
```

Comment:
for easier multi-pattern support. we can perhaps put exclude patterns to a file ?